jupyter-nextflow/nextflow-welcome.html

```diff
@@ -39,17 +39,21 @@ <h1 class=header>Welcome to the Nextflow Workspace</h1>
   </ul>
   <h2 class=header>Get started with Nextflow</h2>
   <p>If you are new to Nextflow, visit <a
-      href=https://www.nextflow.io>nextflow.io</a> for detailed information.
+      href=https://www.nextflow.io>nextflow.io</a> for detailed information. 
+    You can also learn more about using Nextflow in BRH Workspace in the <a
+      href=https://uc-cdis.github.io/BRH-documentation/01-home/>BRH Documentation</a>. 
   </p>
   <p>This workspace is set up to run Nextflow workflows in AWS Batch. Your
-    Nextflow configuration must include the Batch queue, IAM role ARN and work
-    directory that were created for you automatically. The sample configuration
+    Nextflow configuration must include the Batch queue (in our config file, 
+    <code>queue</code>), IAM role ARN (<code>jobRole</code>), and work
+    directory (<code>workDir</code>) that were created for you automatically. The sample configuration
     file will allow you to run simple workflows and can be adapted to your
     needs. To get started:
   </p>
   <ul>
-    <li>If you want your files to persist, first navigate to the "pd" folder.
-    <li>Open a new "Launcher" tab and open a terminal.
+    <li>If you want your your <code>nextflow.config</code> file to persist, first click on the "pd" folder.
```

Comment:
all files will persist, not just the config file.

and i don't think clicking on the folder is enough? you have to be inside the folder. so double clicking

Reply:
This is leading into running the hello world nextflow, where they copy the nextflow config. It feels like this is where they "create" the nextflow config that they can see. Previously, we have emphasized generally about files. here, we should be specific we're talking about the nextflow config file.

Reply:
I added double-click - i need to verify that that's enough

Reply:
@smvgarcia -- where did you add the "double-click" and is the verification done?

Reply:
Also, there are two `your` words in the sentence. Can we fix that well.

Reply:
In a separate commit, apparently. but -- if you move to the PD folder, you cannot run the code there as-is. But, after testing with Sai just now, I see that we just need to add an extra "." before the (so - should be "cp ../data/sample-nextflow-config.txt ./nextflow.config"). I will make this additional commit